core/pubnub_api_types.h

```diff
@@ -261,6 +263,7 @@ enum pubnub_trans {
     /** PAMv3 Grant API transaction sets the permissions for resources and patterns.
       */
     PBTT_GRANT_TOKEN,
+    PBTT_REVOKE_TOKEN,
```

Comment:
@650elx GitHub doesn't let comment on not changed lines, but should our `PUBNUB_USE_GRANT_TOKEN_API` macro be replaced with something more general, because we now use Token API in general? Thoughts?

Reply:
@parfeon maybe PUBNUB_USE_PAM_V3_API?

Reply:
@650elx 👍

Reply:
If we have GRANT and REVOKE as separate flags on portal, this could lose sdk control on the granularity.

Reply:
👌 Thanks for an insight guys